src/track/track.h

```diff
@@ -251,11 +251,12 @@ class Track : public QObject {
 
     // Calls for managing the track's cue points
     CuePointer createAndAddCue();
-    CuePointer findCueByType(Cue::Type type) const; // NOTE: Cannot be used for hotcues.
+    CuePointer findCueByType(mixxx::CueType type) const; // NOTE: Cannot be used for hotcues.
     void removeCue(const CuePointer& pCue);
-    void removeCuesOfType(Cue::Type);
+    void removeCuesOfType(mixxx::CueType);
     QList<CuePointer> getCuePoints() const;
     void setCuePoints(const QList<CuePointer>& cuePoints);
+    void setCuePoints(const QList<mixxx::CueInfo>& cueInfos);
```

Comment:
Rename this method to `importCuePoints()`? Would make it easier to determine which variant is actually invoked without requiring static code analysis.